docs/howtos/convert_pytorch_to_flax.rst

```diff
@@ -0,0 +1,276 @@
+  
+  key = random.PRNGKey(0)
+  x = random.normal(key, (1, 6, 6, 3))
+  
+  j_bn = nn.BatchNorm(momentum=0.9, use_running_average=True)
+  
+  j_out = j_bn.apply(params, x)
+  
+  # [N, H, W, C] -> [N, C, H, W]
+  t_x = torch.from_numpy(np.transpose(np.array(x), (0, 3, 1, 2)))
+  t_out = t_bn(t_x)
+  # [N, C, H, W] -> [N, H, W, C]
+  t_out = np.transpose(t_out.detach().cpu().numpy(), (0, 2, 3, 1))
+  
+  assert np.all(np.abs(j_out - t_out) < 1e-06)
+
+
+
+Average Pooling
+--------------------------------
+
+``torch.nn.AvgPool2d`` and |flax.linen.avg_pool()|_ are compatible when using default parameters.
+However, ``torch.nn.AvgPool2d`` has a parameter ``count_include_pad``. When ``count_include_pad=False``,
+the zero-padding will not be considered for the average calculation. There does not exist a similar
+parameter for |flax.linen.avg_pool()|_. However, we can easily implement a wrapper around the pooling
+operation.
+
+.. |flax.linen.avg_pool()| replace:: ``flax.linen.avg_pool()``
+.. _flax.linen.avg_pool(): https://flax.readthedocs.io/en/latest/_autosummary/flax.linen.avg_pool.html
+
+.. testcode::
+
+   def avg_pool(inputs, window_shape, strides=None, padding='VALID'):
+     """
+     Pools the input by taking the average over a window.
+     In comparison to flax.linen.avg_pool, this pooling operation does not
+     consider the padded zero's for the average computation.
+     Args:
+       inputs: input data with dimensions (batch, window dims..., features).
+       window_shape: a shape tuple defining the window to reduce over.
+       strides: a sequence of `n` integers, representing the inter-window
+           strides (default: `(1, ..., 1)`).
+       padding: either the string `'SAME'`, the string `'VALID'`, or a sequence
+         of `n` `(low, high)` integer pairs that give the padding to apply before
+         and after each spatial dimension (default: `'VALID'`).
+     Returns:
+       The average for each window slice.
+     """
+     assert inputs.ndim == 4
+     assert len(window_shape) == 2
+     
+     # from https://github.com/google/flax/blob/main/flax/linen/pooling.py
+     y = pool(inputs, 0., jax.lax.add, window_shape, strides, padding)
+
+     ones = jnp.ones(shape=(1, inputs.shape[1], inputs.shape[2], 1)).astype(inputs.dtype)
+     counts = jax.lax.conv_general_dilated(ones,
+                                           jnp.expand_dims(jnp.ones(window_shape).astype(inputs.dtype), axis=(-2, -1)),
+                                           window_strides=(1, 1),
+                                           padding=((1, 1), (1, 1)),
+                                           dimension_numbers=nn.linear._conv_dimension_numbers(ones.shape),
+                                           feature_group_count=1)
+     y = y / counts 
+     return y
+
+
+
+
+Transposed Convolutions
+--------------------------------
+
+``torch.nn.ConvTranspose2d`` and |flax.linen.ConvTranspose|_ are not compatible.
```

Comment:
This paragraph will make the reader wonder how a `torch.nn.ConvTranspose2d` should be converted. Do you know how?

Reply:
Thanks for adding the reference. I created #1872 to track this.